### DISCUSSION TITLE
NEVER power the RAD from the computer and externally at the same time.

### DISCUSSION THREAD

**Discussion by Pinks64** [frntc/RAD#55] [Q&A]
Hi I just accidentally power on c64 with plugged psu for rpi in wall. Just for seconds, for now i dont see any damage in operating. Sonic is loading and hungs on this machine but all others things works fine. On another c64 unit everything is working fine.

What can go wrong in this situation? CPU will die? CIA? Is there any way to add circuit to secure c64 and rad?

**Comment by futureretrofusion** [comment]
I've had a few accidents like that but immediately stopped and never
noticed an issue, though it can result in damage.

If you are using pizero.no issues just powering from the c64 cart port and
forget about external PSU? Pi3a+ should be good for powering up on it own
also.

I'm assuming the damage can be caused by power coming from 2 separate
sources could cause overvoltage if they are not on the same ground?
